src/test/java/jenkins/plugins/office365connector/FormUtilsTest.java

```diff
@@ -0,0 +1,50 @@
+package jenkins.plugins.office365connector;
+
+import hudson.util.FormValidation;
+import jenkins.plugins.office365connector.util.FormUtils;
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+
+public final class FormUtilsTest {
+
+    private static final String NOT_VALID = "demo";
+    private static final String NOT_VALID_URL = "hpp://demo";
+    private static final String NOT_VALID_VARIABLE_REFERENCE = "$$$$demo";
+    private static final String VALID_VARIABLE_REFERENCE = "$demo";
+    private static final String VALID_URL = "http://demo";
+
+    @Test
+    public void validUrl() {
+        FormValidation result = FormUtils.validateUrl(VALID_URL);
+        Assert.assertThat(result.kind, is(FormValidation.Kind.OK));
```

Comment:
so once you move above to local variable you can easily switch to BDD approach and divide this piece of code into
// given
// when
// then